war/src/main/less/pluginSetupWizard.less

```diff
@@ -428,7 +428,7 @@
 						content: '';
 						left: 2px;
 						top: 3px;
-						background: @badge-color;
+						background: var(--badge-color);
```

Comment:
Is this variable declared anywhere? 

I wouldn't apply the styles to the setup wizard. Right know they are on their own and can behave weirdly if the variables are mixed.